src/openssl/algorithm.cr

```diff
@@ -0,0 +1,34 @@
+require "openssl"
+
+module OpenSSL
+  # Indicates the possible hash algorithms for `OpenSSL` digest operations.
+  enum Algorithm
+    MD4
+    MD5
+    RIPEMD160
+    SHA1
+    SHA224
+    SHA256
+    SHA384
+    SHA512
+
+    # The internal bindings to the `LibCrypto` digest operations sometimes require a hash algorithm
```

Comment:
Can this be moved below the `This method will return ...` section? It's rather meta information and knowing what this method actually returns is probably more important.

Reply:
Like reversing the order of the paragraphs?
I'll do this

Reply:
Keep in mind that my english is poor, so I'd rather copy&paste any better phrasing.

Reply:
Yes reversing the order of the two sections:
```cr
    # Returns the appropriate equivalent hash algorithm that corresponds to the
    # current enum value.
    #
    # The internal bindings to the `LibCrypto` digest operations sometimes require a hash algorithm
    # implementation to be passed as one of the arguments.
```

Reply:
Nice. I've done this.